MyHashSet.java

```diff
@@ -0,0 +1,60 @@
+// Time Complexity : All operation are O(1) as we are using O(n) space
+// Space Complexity : O(n) as we are using double hashing technique with one unique space for every input
+// Did this code successfully run on Leetcode : Yes
+// Any problem you faced while coding this : No
+
+
+// Your code here along with comments explaining your approach
+
+class MyHashSet {
+
+    /** Initialize your data structure here. */
+    boolean[][] arr;
+    public MyHashSet() {
+        arr = new boolean[1000][];
+    }
+
+    public int bucket(int key){
+        return key%1000;
+    }
+
+    public int bucketItem(int key){
+        return key/1000;
+    }
+
+    public void add(int key) {
+        int bucket = bucket(key);
```

Comment:
try to include some comments, helps you when you revisit in future for the logic